packages/lesswrong/lib/crud/withMulti.ts

```diff
@@ -425,3 +432,55 @@ export function useMulti<
     limit: effectiveLimit,
   };
 }
+
+/**
+ * Load a list of documents using a view, callback-style. Should only be called
+ * from event handlers and useEffect; calling this from directly inside a render
+ * function will not work. In the common case where a React component is loading
+ * data based on its props, you should use useMulti instead.
+ *
+ * Beware: While the signature of this function is straightforward, the
+ * circumstances in which you would actually have to use it are likely to
+ * involve tricky concurrency issues.
+ *
+ * Returns the list of results, or throws an exception if there's an error such
+ * as a network error. The number of results will be at most the number given as
+ * `limit`, but may be less if there are fewer results than that available, or
+ * if something was removed from the results list by collection-permissions
+ * filtering.
+ *
+ * TODO: Check that the above about error handling is actually true.
+ */
+export const loadMulti = async <
+  FragmentTypeName extends keyof FragmentTypes,
+  CollectionName extends CollectionNameString = CollectionNamesByFragmentName[FragmentTypeName]
+>({terms, limit, client, fragmentName, collectionName, fetchPolicy}: {
+  terms: ViewTermsByCollectionName[CollectionName],
+  limit: number,
+  client: ApolloClient<any>,
```

Comment:
Can we default using the regular client, and leave it as an optional parameter if we ever need to pass in a different one, such as for crossposting?   (And if we don't, then we can skip the parameter entirely for now)

Reply:
We can't, because this isn't a hook so it doesn't have access to React context.